Bottom: 291306aaa737ed58b7032b342c9a0a5cfea62168
Top:    99c9c1e9913c91ea420dd1191ac82f4384363bc0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:25 -0500

Remove libicu52 from Dockerfile
    
    Newest upstream versions of php:5.6-apache are based on debian stretch and
    use libicu57, which is already installed. Besides this, libicu-dev would
    already install libicuXX.


---
diff --git a/docker/docker-mantisbt/Dockerfile b/docker/docker-mantisbt/Dockerfile
index b59a97c7..7db874b3 100644
--- a/docker/docker-mantisbt/Dockerfile
+++ b/docker/docker-mantisbt/Dockerfile
@@ -8,23 +8,22 @@ MAINTAINER Joseph Lutz <Joseph.Lutz@novatechweb.com>
 
 ENV MANTISBT_VERSION 1.2.19
 
+#         ttf-mscorefonts-installer
+
 RUN sed -i 's| main$| main contrib non-free|' /etc/apt/sources.list \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         libcurl3 \
-        libicu52 \
         libjpeg62-turbo \
         libmcrypt4 \
         libpq5 \
-        ttf-mscorefonts-installer \
-        \
         libcurl4-openssl-dev \
         libfreetype6-dev \
         libicu-dev \
         libjpeg62-turbo-dev \
         libldap2-dev \
         libmcrypt-dev \
-        libpng12-dev \
+        libpng-dev \
         libpq-dev \
     && docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu \
     && docker-php-ext-configure gd --with-png-dir=/usr --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
